src/engine/enginemaster.cpp

```diff
@@ -155,14 +171,17 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
     m_pKeylockEngine->set(pConfig->getValueString(
             ConfigKey(group, "keylock_engine")).toDouble());
 
+    // TODO: Make this read only but let EngineMasterTest enable it
```

Comment:
Because the master mixxx goes to recording and broadcasting as well. In Mixxx 2.0 the user can choose if he needs the master mix and enable or disable it manually on demand. 
If a user feets the deck outputs to an external mixer and has no recording or broadcasting, he can disable the master mix.